test/functional/p2p_leak.py

```diff
@@ -96,15 +98,25 @@ def set_test_params(self):
         self.num_nodes = 1
         self.disable_mocktime = True
```

Comment:
missing: https://github.com/bitcoin/bitcoin/pull/21345/files#diff-6ac274756f81c40c6dea972924d2e581e35c8e2c6f3864db8e37a0a0995ca3e6R101?

Reply:
yeah, but its replacement is added in the next backport (22153) so it's kind of ok-ish